R/mse_estimation.R

```diff
@@ -143,19 +147,30 @@ mse_estim <- function(framework,
     framework$threshold <-
       framework$threshold(y = pop_income_vector)
   }
+
+  if(is.null(framework$aggregate_to_vec) != TRUE){
+    N_dom_pop_tmp <- framework$N_dom_pop_agg
+    pop_domains_vec_tmp <- framework$aggregate_to_vec
+    pop_weights_vec <- framework$pop_data[[framework$pop_weights]]
```

Comment:
pop_weights could be NULL here, couldnt it be?

Reply:
corrected and proffed

Reply:
I am still not really sure about this part. Is this really correct, that I only check if aggregate_to_vec is not NULL and then I use the population weights, and when aggregate_to_vec is NULL I do not use population weights? Aren't these four combinations? I can have aggregate_to is not NULL and pop_weights is not null, and both could be null or only one is null and the other is not?

Reply:
Du hast rcht. Dankschön. Ist korrigiert.